.env.example

```diff
@@ -6,6 +6,8 @@ ADD_POINT_TAG_ID = 1258801833191669802 # ID for task completed tag.
 ADD_BOOSTED_POINT_TAG_ID = 1263873487953592381 # ID for the tag that adds boosted points
 DISCORD_QA_ROLE_ID=1140734540139204800
 DISCORD_QA_CHANNEL_ID=qa-request
+ADMINISTRATOR_TAG_ID=1140734540139204800
+ADMIN_POINT_REQUEST_CHANNEL=1270770663048613983
```

Comment:
```suggestion
ADMINISTRATOR_TAG_ID=1014267063788908585
ADMIN_POINT_REQUEST_CHANNEL=1310614733803618416
```

Reply:
Adding as default the ones that we have on our prod server

Reply:
Complete